library/src/main/java/com/novoda/downloadmanager/LiteDownloadBatchStatus.java

```diff
@@ -149,6 +150,12 @@ public void markAsWaitingForNetwork(DownloadsBatchPersistence persistence) {
         updateStatusAsync(status, persistence);
     }
 
+    @Override
+    public void markAsNotificationSeen(DownloadsNotificationSeenPersistence persistence) {
```

Comment:
Following the scheme of the others in this class now that we need to update both this status and the underlying DB version.